### PR TITLE
added prerelease notes

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -24,7 +24,52 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
+
+
+## 1.0.1010-prerelease
+
+Release Date: September 13, 2021  
+
+[NuGet package for WebView2 SDK 1.0.1010-prerelease][NuGetGallery1.0.1010-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 94.0.1010.0 or higher.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:36]:
```suggestion
For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 95.0.1010.0 or higher.
```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -24,7 +24,52 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+*   Fixed bugs where an `IDispatch` objects were returned as `IUnknown`.
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+*   `IsSwipeNavigationEnabled`.
+*   `BrowserProcessExited`.
+*   `OpenBrowserTaskManager`.
 
 ## 1.0.961.33
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:74]: Can we change the minimum version for 1.0.961.33 package to 93.0.961.**44**, the minimum version that includes a runtime client certificate api change?

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -24,25 +24,70 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
 
 Release Date: September 8, 2021  
 
 [NuGet package for WebView2 SDK 1.0.961.33][NuGetGallery1.0.961.33]
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 93.0.961.33 or higher.
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 93.0.961.33 or higher.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:80]:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 93.0.961.44 or higher.
```
@jasonstephen15 - a change per Jessica that you can commit

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:74]: I managed to add that comment as a Suggestion on line 80

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -24,25 +24,70 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 
 To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.
 
-To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
+To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
+
+
+## 1.0.1010-prerelease
+
+Release Date: September 13, 2021
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:32]:
```suggestion
Release Date: September 14, 2021  
```

**Comment by jasonstephen15** [comment]
@Reezaali @mikehoffms ready to merge!

**Pull request merged**